docs/cli.md

```diff
@@ -49,7 +49,7 @@ Pulls service images.
 
 ### rm
 
-Removes stopped service containers.
+Removes stopped service containers. _This removes the containers completely, you will lose any data that was contained on them_.
```

Comment:
Removes stopped service containers. Ay data stored in the container's local storage is lost when a container is removed.